assets/_scss/components/_forms.scss

```diff
@@ -23,7 +23,7 @@ form {
     margin-bottom: 1.5em;
   }
 
-  input[name="password"] {
+  input[name="password"], input[name="confirmPassword"] {
```

Comment:
> Also, does adding bottom margin to these two password inputs work in different configurations of form elements? Such as when you could have password inputs before another form element?
Yes, I just tried and since we have a default `margin-top: 3rem` applied to labels which go before inputs and since margins collapse it works fine. @msecret 

I think that a better method in the long run would be adding `margin-top` to the show password link. But we can just add a note about this to change later.